auth-web/package.json

```diff
@@ -25,7 +25,7 @@
     "@vue/compiler-dom": "^3.2.45",
     "@vue/composition-api": "^1.7.1",
     "axios": "^1.6.0",
-    "fas-ui": "github:bcgov/fas-ui#v1.1.4",
```

Comment:
Old version was causing a npm error that could only be replicated in CI and Dockerfile